Aptoide/appcoins-wallet-android
# [APPC-1134] add rtl support on the on boarding viewpager

**What does this PR do?**

   This PR implements the RTL support for the onboarding viewpager

**Database changed?**

No

**Where should the reviewer start?**

- [ ] OnboardingActivity.kt

**How should this be manually tested?**
For LTR and for RTL languages open the wallet for the first time and check if the viewpager behaviour matches the orientation type.
 
  Flow on how to test this or QA Tickets related to this use-case: [APPC-1134](https://aptoide.atlassian.net/browse/APPC-1134)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-1134](https://aptoide.atlassian.net/browse/APPC-1134)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)




**Code Review Checklist**
